src/index.js

```diff
@@ -164,7 +175,7 @@ class DBLAPI extends EventEmitter {
 
   /**
    * Gets stats from a bot.
-   * @param {string} id The ID of the bot you want to get the stats from.
+   * @param {string} id=client.user.id The ID of the bot you want to get the stats from.
```

Comment:
You missed this

Reply:
That's still viable no? It does still default to the client.user.id

Reply:
I don't know if that line's valid but I doubt it